bindings/ldk_node.udl

```diff
@@ -72,7 +72,7 @@ interface LDKNode {
 	[Throws=NodeError]
 	PaymentHash send_payment_using_amount([ByRef]Bolt11Invoice invoice, u64 amount_msat);
 	[Throws=NodeError]
-	PaymentHash send_spontaneous_payment(u64 amount_msat, PublicKey node_id);
+	PaymentHash send_spontaneous_payment(u64 amount_msat, PublicKey node_id, sequence<TlvEntry> custom_tlvs);
```

Comment:
Should `custom_tlvs` be optional?

Reply:
It's a sequence, so an empty sequence unambiguously signals "no TLVs", and further wrapping it into an `Option` seemed like over-engineering to me :)

Reply:
Sounds good, thanks!